src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -276,8 +277,7 @@ def __call__(self, event, organization=None):
 class PullRequestEventWebhook(Webhook):
     # https://developer.github.com/v3/activity/events/types/#pullrequestevent
     def __call__(self, event, organization):
-        action = event['action']
-        # TODO(maxbittker) handle is_apps correctly
+        # TODO(maxbittker) handle is_apps correctly (What does this comment mean?)
```

Comment:
basically, we have two providers (`github` and `github_apps`) and this code doesn't work with github apps right now (but neither does the rest of it AFAIK)